### DISCUSSION TITLE
Update container description on ghcr.io

### DISCUSSION THREAD

**Discussion by miaucl** [community/community#25664] [Actions]
<p>While it is possible to update the <code>README.md</code> of <em>npm</em> packages and <em>docker</em> containers for the repo-specific Github Packages directly from the web interface, I could not find a possibility to do so for the organisation-wide container registry.</p>
<p>Q1: Is the <code>README.md</code> pulled from the repo kept up-to-date or does it need to be updated manually through github actions?<br>
Q2: Is there a possibility to choose something else as the description instead of the <code>README.md</code>?</p>
<p>Thank you!</p>

**Comment by jcansdale** [comment, marked as answer]
<p>Hi <a class="mention" href="/miaucl">@miaucl</a>,</p>
<blockquote>
<p>Q1: Is the <code>README.md</code> pulled from the repo kept up-to-date or does it need to be updated manually through github actions?</p>
</blockquote>
<p>It is kept up-to-date with the linked repository.</p>
<blockquote>
<p>Q2: Is there a possibility to choose something else as the description instead of the <code>README.md</code> ?</p>
</blockquote>
<p>You can’t choose an alternative <code>README.md</code>, but you can specify a description using a <code>LABEL</code> in your <code>Dockerfile</code>. It would look something like this:</p>
<pre><code class="lang-auto">LABEL org.opencontainers.image.description="Hello, World!"
</code></pre>
<p>You can see the description here:<br>
<a href="https://github.com/orgs/github-packages-examples/packages/container/ghcr-publish/1556270#js-pjax-container:~:text=About%20this%20version" rel="noopener nofollow ugc">About this version</a></p>

**Comment by Niek** [comment]
<p>I have specified a <code>org.opencontainers.image.description</code> in my Dockerfile, but the packages page still shows the README of the linked repo (from <code>org.opencontainers.image.source</code>). Is there any way to override this or to point to a different README?</p>
